frontend/src/components/AssessmentCard.js

```diff
@@ -0,0 +1,69 @@
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import { LinearProgress } from '@mui/material';
+
+import QuestionWrapper from './QuestionWrapper';
+import { boxStyles } from './boxStyles';
+import { useEffect, useState } from 'react';
+
+const AssessmentCard = ({ completeAssessment, responses, setResponses }) => {
+  const [activeQuestionIndex, setActiveQuestionIndex] = useState(0);
+  const [assessmentData, setAssessmentData] = useState([]);
+
+  function fetchAssessmentData() {
+    fetch('http://localhost:3002/api/assessments/data')
```

Comment:
Ideally, this fetch would be to `api/assessments/data` but I couldn't quite figure it out in the time I had. I tried adding the proxy to my package.json but that seems to cause the app to not load. I'm sure it's some sort of networking issue.